### PR TITLE
added addNewLocation scene

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/CameraView.js [hunk 1]
```diff
@@ -161,79 +151,110 @@ class CameraView extends Component {
...
+          />
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.btn_flash}
+          onPress={this.handleFlash.bind(this)}>
+          <Image
+            style={{
+              height: 48, width: 48,
+              left: Dimensions.get('window').width / 3 - 48 - 32}}
+            source={require('../resources/camera/icon_flash.png')}
```
Comment (ovekyc) [inline shared/components/CameraView.js]: ditto

File: shared/components/CameraView.js [hunk 2]
```diff
@@ -161,79 +151,110 @@ class CameraView extends Component {
...
+
+  renderBottomAfter() {
+    return (
+      <View style={styles.bottom}>
+        <TouchableOpacity
+          style={styles.btn_again}
+          onPress={this.takeAgain.bind(this)}>
+          <Image
+            style={{height: 48, width: 48}}
+            source={require('../resources/camera/icon_camera_again.png')}
```
Comment (ovekyc) [inline shared/components/CameraView.js]: ditto

File: shared/components/Create.js [hunk 3]
```diff
@@ -67,90 +122,361 @@ const styles = StyleSheet.create({
...
+      addingNewLocation: true
+    })
+  }
+
+  handleAddExistingLocation() {
+    //todo: implement adding photo to an existing location
+  }
+
+  convertMonth(m) {
+    switch (m) {
```
Comment (ovekyc) [inline shared/components/Create.js]: It also could be 
const month = ['Jan', 'Feb', ...];
return month[m-'1'];

File: shared/components/Create.js [hunk 4]
```diff
@@ -67,90 +122,361 @@ const styles = StyleSheet.create({
...
+    case '10': return 'Oct.';
+    case '11': return 'Nov.';
+    case '12': return 'Dec.';
+    default: return '';
+    }
+  }
+
+  handleOnDateChange(datetime) {
+    this.setState({dateStart: datetime});
+    let a = new Date(datetime);
```
Comment (ovekyc) [inline shared/components/Create.js]: Please use const

File: shared/components/Create.js [hunk 5]
```diff
@@ -67,90 +122,361 @@ const styles = StyleSheet.create({
...
+    let txtDate = this.convertMonth(
```
Comment (ovekyc) [inline shared/components/Create.js]: You can use const and template
`${a.format('MM'))} ${a.format('DD')}, .....`

File: shared/components/Create.js [hunk 6]
```diff
@@ -67,90 +122,361 @@ const styles = StyleSheet.create({
...
+            </View>
+          </View>
+        </TouchableOpacity>
+        {this.renderAroundLocations()}
+      </View>
+    )
+  }
+
+  getDistanceFromLatLonInKm(lat1,lon1) {
+    let lat2 = this.props.currentLocation.latitude;
```
Comment (ovekyc) [inline shared/components/Create.js]: Please use const and edit variable names

File: shared/components/Create.js [hunk 7]
```diff
@@ -67,90 +122,361 @@ const styles = StyleSheet.create({
...
+  }
+
+  renderBtnBefore() {
+    return (
+      <TouchableOpacity
+        style={styles.btn_before}
+        onPress={this.handleBefore.bind(this)}>
+        <Image
+          style={{height:24, width: 24}}
+          source={require('../resources/camera/btn_before.png')}
```
Comment (ovekyc) [inline shared/components/Create.js]: ditto

File: shared/components/CameraView.js [hunk 8]
```diff
@@ -161,79 +151,110 @@ class CameraView extends Component {
...
   }
 
+  renderBtnClose() {
+    return (
+      <TouchableOpacity
+        style={styles.btn_close}
+        onPress={this.handleClose.bind(this)}>
+        <Image
+          style={{height:24, width: 24}}
+          source={require('../resources/camera/btn_close.png')}
```
Comment (ochanje210) [inline shared/components/CameraView.js]: solved

File: shared/components/CameraView.js [hunk 9]
```diff
@@ -161,79 +151,110 @@ class CameraView extends Component {
...
+
+  renderBottom() {
+    return (
+      <View style={styles.bottom}>
+        <TouchableOpacity
+          style={styles.btn_camera_switch}
+          onPress={this.switchType.bind(this)}>
+          <Image
+            style={{height: 48, width: 48, left: 32}}
+            source={require('../resources/camera/icon_camera_switch.png')}
```
Comment (ochanje210) [inline shared/components/CameraView.js]: solved

File: shared/components/CameraView.js (hunk 1 above)
Comment (ochanje210) [inline shared/components/CameraView.js]: solved

File: shared/components/CameraView.js (hunk 2 above)
Comment (ochanje210) [inline shared/components/CameraView.js]: solved

File: shared/components/Create.js (hunk 7 above)
Comment (ochanje210) [inline shared/components/Create.js]: solved

File: shared/components/Create.js (hunk 5 above)
Comment (ochanje210) [inline shared/components/Create.js]: thank you!

File: shared/components/Create.js (hunk 4 above)
Comment (ochanje210) [inline shared/components/Create.js]: followed!

**Comment by ovekyc** [comment]
lgtm

**Comment by ochanje210** [comment]
LGTM

**Pull request merged**